js/quiz.js

```diff
@@ -0,0 +1,42 @@
+
+//convert to bootstrap
+
+function populatePage (inventoryData) {
+	for (i = 0; i < inventoryData.length; i++) {
+
+		// Create the elements
+		var div = document.createElement("ARTICLE");
+		var header = document.createElement("HEADER");
+		var section = document.createElement("SECTION");
+		var footer = document.createElement("FOOTER");	
+
+		// Sort through the array 
+		getinfo = inventoryData[i];
+		div.classList.add("car", "border");
+		
+		div.id = `car${i}`;
+		header.innerHTML  = getinfo.year + " " + getinfo.make + " " + getinfo.model  ; 
+		section.innerHTML = "$" + getinfo.price;
+		footer.innerHTML  = getinfo.description;
+
+		// Append and output
+		div.appendChild(header);
+		div.appendChild(section);
+		div.appendChild(footer);
+		output.appendChild(div);
+	
+		document.getElementById(`car${i}`).addEventListener("click", function() {
+			this.classList.add("borderColor");
```

Comment:
Good job getting the click event added. The next step for this function is to make it also remove the `borderColor` class from any card that already has that class, so that when you click a card, then click a second card, only the second card has the border and background color. In other words, only one card at a time should have that class on it. See if you can figure out how to make that happen. You'll need to check the DOM for the presence of `borderColor` first thing in this function and remove it before adding it to the clicked card.

Then, later, when you add your event listener for the text input, you can target the card with that `borderColor` class on it as the one where you want to insert the text from the input. 

Also, instead of the anonymous function you have here as your callback, you can reference the method you've defined, `addBorder` in `border.js`